Payments/Operation/04_ManageCancellationsChargebacks.md

```diff
@@ -0,0 +1,89 @@
+[!!LIST (Cancellations and chargebacks](../UserInterface/05a_ListCancellationsChargebacks.md)
+
+# Monitor cancellations and chargebacks
+
+In the context of a payment transaction a chargeback is a retroactive cancellation of a capture. It is initialized by the customer, for example, if he or she withdraws the payment. A cancellation is a revocation of the reservation of a product that has not yet been delivered. It is also initialized by the customer.    
+
+> [Info] The most payment service providers create a new reference, when a payment has been cancelled or charged back by the customer. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.  
```

Comment:
```suggestion
> [Info] The most payment service providers create a new reference when a payment has been cancelled or charged back by the customer. However, the most payment service providers support an original reference ID that you can use to search for a specific case in the whole payment process.  
```
"The most payment service providers create a new reference..." -> Vielleicht die Info so immer anfangen, auch bei den anderen Dateien (anstatt "You cannot follow up... -> positive Aussage hört sich besser an)

Reply:
sagt aber deutlicher, was Sache ist